tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 85.25, "AMD": 84.50, "ARM": 82.72}
+COVERAGE_DICT = {"Intel": 85.04, "AMD": 84.27, "ARM": 82.98}
```

Comment:
x86 coverage dropping while the ARM one is rising. Maybe they are trending towards the a mean value?